readme.md

```diff
@@ -201,15 +211,15 @@ Once installation is working, take a look at the existing code to make sure you
 * **Question:** What happens? Why?
 
 * **Your Answer:**
-
+the bearer token is split and verified through comparing with the secret key.  Then the user id is pulled from the payload
```

Comment:
- 404 Error received with message, `You are not authorized to access this route`. this is because an auth token was not included with the HTTP request to provide "authorization".